src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -0,0 +1,57 @@
+import React, {
+  createRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../icon/icon.jsx';
+import iconCaretDown from '../../svgs/icon-caret-down.svg';
+import iconCaretDownDisabled from '../../svgs/icon-caret-down-disabled.svg';
+import TagList from '../tag-list/tag-list.jsx';
+import Tag from '../tag/tag.jsx';
+import styles from './custom-field-input-multiple-choice.css';
+
+function CustomFieldInputMultipleChoice(props) {
+  const refs = props.value.map(() => createRef());
```

Comment:
I think this is better?

```suggestion
  const refs = props.value.map(() => useRef());
```